vllm/config.py

```diff
@@ -419,6 +419,8 @@ class ModelConfig:
     - "transformers" will use the Transformers model implementation."""
     override_attention_dtype: Optional[str] = None
     """Override dtype for attention"""
+    enforce_include_usage: bool = False
+    """Enforce including usage on every request."""
```

Comment:
Actually this seems to be strictly frontend, we proably don't want it to be in model_config here

Reply:
I think it is better to include it in the CLI, see the `enable-prompt-tokens-details` options.

maybe we can also have a `enable-force-include-usage` here.

Reply:
fwiw I think we should also add an options to the request level, where some might want certain request to have usage, some requests without. Given that usage will affect throughput quite a lot.